frontend/src/App.css

```diff
@@ -1,4 +1,8 @@
 /* App.css */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 .app {
   display: flex;
```

Comment:
should be able to remove this line now that you have the `flex` class.

In general, the hope with tailwind is that we can remove _all_ of these CSS styles, or at least 95% of them. Can probably do that in future PRs though